pkg/driver/nydus/packer/mount.go

```diff
@@ -0,0 +1,109 @@
+// Copyright Project BuildKit Authors
```

Comment:
Should have a reference to the original file

Reply:
Thanks, added a comment for this.